src/bejeweled/game/GameScene.java

```diff
@@ -50,7 +55,24 @@ public GameScene(Group root) {
 		saveButton.setOnAction(new EventHandler<ActionEvent>() {
 			@Override
 			public void handle(ActionEvent e) {
-				//Write gamestate to file to save the game.
```

Comment:
Probably wanna put this in a different method, maybe within a saveclass or something?